config/galaxy.ini.sample

```diff
@@ -203,7 +203,8 @@ paste.app_factory = galaxy.web.buildapp:app_factory
 # Pass debug flag to conda commands.
 #conda_debug = False
 # conda channels to enable by default (http://conda.pydata.org/docs/custom-channels.html)
-#conda_ensure_channels = conda-forge,r,bioconda,iuc
+# the recommended channel order is the one from BioConda (https://github.com/bioconda/bioconda-recipes/blob/master/config.yml#L8)
+#conda_ensure_channels = bioconda,r,defaults,conda-forge,iuc
```

Comment:
If `iuc` is only for old recipes, its position shouldn't matter, but if we are ever going to need to use it for some "ugly hack" it needs to be first.

Has any one else opinions here?